convert2rhel/actions/conversion/set_efi_config.py

```diff
@@ -0,0 +1,243 @@
+                logger.debug(
+                    "The %s file does not exist in %s folder. Copying skipped."
+                    % (os.path.basename(src_file), CENTOS_EFIDIR_CANONICAL_PATH)
+                )
+                continue
+            # Skip already existing file in destination directory
+            dst_file = os.path.join(RHEL_EFIDIR_CANONICAL_PATH, os.path.basename(src_file))
+            if os.path.exists(dst_file):
+                logger.debug(
+                    "The %s file already exists in %s folder. Copying skipped."
+                    % (os.path.basename(src_file), RHEL_EFIDIR_CANONICAL_PATH)
+                )
+                continue
+
+            logger.info("Copying '%s' to '%s'" % (src_file, dst_file))
+
+            try:
+                shutil.copy2(src_file, dst_file)
+            except (OSError, IOError) as err:
+                # IOError for py2 and OSError for py3
+                self.set_result(
+                    level="ERROR",
+                    id="GRUB_FILES_NOT_COPIED_TO_BOOT_DIRECTORY",
+                    title="GRUB files have not been copied to boot directory",
+                    description=(
+                        "I/O error(%s): '%s'. Some GRUB files have not been copied to /boot/efi/EFI/redhat."
+                        % (err.errno, err.strerror)
+                    ),
+                )
+
+
+class RemoveEfiCentos(actions.Action):
+    id = "REMOVE_EFI_CENTOS"
+    dependencies = ("COPY_GRUB_FILES",)
+
+    def run(self):
+        """Remove the /boot/efi/EFI/centos/ directory when no UEFI files remains.
+
+        The centos/ directory after the conversion contains usually just grubenv,
+        grub.cfg, .. files only. Which we copy into the redhat/ directory. If no
+        other UEFI files are present, we can remove this dir. However, if additional
+        UEFI files are present, we should keep the directory for now, until we
+        deal with it.
+        """
+        super(RemoveEfiCentos, self).run()
+
+        if systeminfo.system_info.id != "centos":
+            logger.debug("Did not perform removal of EFI files - only related to CentOS Linux.")
+            # nothing to do
+            return
+        try:
+            os.rmdir(CENTOS_EFIDIR_CANONICAL_PATH)
+        except (OSError, IOError) as err:
+            warning_message = (
+                "Failed to remove the {dir} directory as files still exist."
+                " During conversion we make sure to copy over files needed to their RHEL counterpart."
+                " However, some files we didn't expect likely exist in the directory that needs human oversight."
+                " Make sure that the files within the directory is taken care of and proceed with deleting the directory"
+                " manually after conversion. We received error: '{err}'."
+            ).format(dir=CENTOS_EFIDIR_CANONICAL_PATH, err=err)
+                dir=CENTOS_EFIDIR_CANONICAL_PATH, err=err
+            )
```

Comment:
```suggestion
            warning_message = (
                "Failed to remove the {dir} directory as files still exist. During conversion we make sure to copy "
                "over files needed to their RHEL counterpart. However, some files we didn't expect likely exist in the "
                "directory that needs human oversight. Make sure that the files within the directory is taken care of "
                "and proceed with deleting the directory manually after conversion. We received error: '{err}'."
            ).format(dir=CENTOS_EFIDIR_CANONICAL_PATH, err=err)
```

Reply:
I would say the `\` are needed there in the ends of the lines, when I was testing that it wasn't working correctly without them (prints just the first line without formatting)

Reply:
Could you share your test? Reproducing this in a python shell works correctly

Reply:
```python
>>> CENTOS_EFIDIR_CANONICAL_PATH="test"
>>> err="test2"
>>> warning_message = (
                "Failed to remove the {dir} directory as files still exist. During conversion we make sure to copy "
                "over files needed to their RHEL counterpart. However, some files we didn't expect likely exist in the "
                "directory that needs human oversight. Make sure that the files within the directory is taken care of "
                "and proceed with deleting the directory manually after conversion. We received error: '{err}'."
            ).format(dir=CENTOS_EFIDIR_CANONICAL_PATH, err=err)
>>> logger.info(warning_message)
INFO:__main__:Failed to remove the test directory as files still exist. During conversion we make sure to copy over files needed to their RHEL counterpart. However, some files we didn't expect likely exist in the directory that needs human oversight. Make sure that the files within the directory is taken care of and proceed with deleting the directory manually after conversion. We received error: 'test2'.
>>> print(warning_message)
Failed to remove the test directory as files still exist. During conversion we make sure to copy over files needed to their RHEL counterpart. However, some files we didn't expect likely exist in the directory that needs human oversight. Make sure that the files within the directory is taken care of and proceed with deleting the directory manually after conversion. We received error: 'test2'.
```

Reply:
I see, the difference is the `()`, so `()` or the `\` is needed. Sorry